go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/Azure/go-autorest/autorest v0.9.0
 	github.com/Azure/go-autorest/autorest/date v0.2.0
 	github.com/google/uuid v1.3.1
-	github.com/microsoft/moc v0.11.0-alpha.37
+	github.com/microsoft/moc v0.13.1-0.20231218211421-786409bcad4d
```

Comment:
Is this a published package version?

Reply:
We are using the most recent release version now.